mmv1/third_party/terraform/provider/provider_mmv1_resources.go.tmpl

```diff
@@ -126,6 +126,7 @@ var handwrittenDatasources = map[string]*schema.Resource{
 	"google_container_registry_repository":             containeranalysis.DataSourceGoogleContainerRepo(),
 	"google_dataproc_metastore_service":                dataprocmetastore.DataSourceDataprocMetastoreService(),
 	"google_datastream_static_ips":                     datastream.DataSourceGoogleDatastreamStaticIps(),
+	"google_datastream_private_connection":			 	datastream.DataSourceGoogleDatastreamPrivateConnection(),
```

Comment:
Can you align datastream.DataSourceGoogleDatastreamPrivateConnection() with those in other lines? Thanks.